docs/products/kafka/kafka-connect/howto/kafka-connect-debezium-tls-pg.md

```diff
@@ -0,0 +1,213 @@
+    avn service create <kafka_cluster_name> \
+      --service-type kafka \
+      --cloud <cloud_provider_and_region> \
+      --plan <plan> \
+      --project <project_name>
+    ```
+
+    :::note
+    Ensure [topic auto-creation](/docs/products/kafka/howto/create-topics-automatically)
+    is enabled to automatically generate required Apache Kafka topics. If disabled,
+    manually create topics before starting the connector.
+    :::
+
+1. Create an Apache Kafka Connect service and configure it to communicate with your
+   Aiven for Apache Kafka service:
+
+    ```bash
+    avn service create <kafka_connect_name> \
+      --service-type kafka_connect \
+      --cloud <cloud_provider_and_region> \
+      --plan <plan> \
+      --project $PROJECT
+    ```
+
+1. Create an external PostgreSQL integration endpoint to represent your PostgreSQL
+   database. Use the following JSON configuration and replace the placeholders:
+
+    ```bash
+    INTEGRATION_CONFIG=$(cat <<-END
+    {
+      "ssl_mode": "verify-ca",
+      "host": "<postgresql_host>",
+      "port": <postgresql_port>,
+      "user": "<postgresql_user>",
+      "ssl_client_certificate": "$(cat /path/to/your/client-cert.pem)",
+      "ssl_root_cert": "$(cat /path/to/your/ca.pem)",
+      "ssl_client_key": "$(cat /path/to/your/client-key.pem)"
+    }
+    END
+    )
+
+    avn service integration-endpoint-create \
+	   --project $PROJECT \
+	   --endpoint-name external_postgresql \
+	   --user-config "$INTEGRATION_CONFIG"
+
+   ```
+
+1. Retrieve endpoint ID of the integration endpoint you just created using this command:
+
+   ```bash
+   INTEGRATION_ENDPOINT_ID=$(
+	   avn service integration-endpoint-list --project $PROJECT \
+	   | grep external_postgresql \
+	   | awk '{print $1}'
+	  )
+   ```
+
+1. Connect PostgreSQL endpoint to Apache Kafka Connect:
+
+   ```bash
+   avn service integration-create \
+     --project $PROJECT \
+     --integration-type kafka_connect_postgresql \
+     --source-endpoint-id $INTEGRATION_ENDPOINT_ID \
+     --dest-service <kafka_connect_name>
+   ```
+
+1. Create the Debezium connector configuration to monitor your PostgreSQL database.
+   Replace the placeholders with your PostgreSQL and Apache Kafka Connect information:
+
+   ```bash
+   CONNECTOR_CONFIG=$(cat <<-END
+   {
+     "name": "debezium-postgres-connector",
+     "connector.class": "io.debezium.connector.postgresql.PostgresConnector",
+     // Omit "database.hostname", "database.port", and "database.user" if provided
+     by external integration
+     "database.password": "<postgresql_password>",
+     "database.dbname": "<database_name>",
+     "database.server.name": "<kafka_connect_server_name>",
+     "plugin.name": "pgoutput",
+     "publication.name": "debezium_publication",
+     "publication.autocreate.mode": "all_tables",
+     "endpoint_id": "$INTEGRATION_ENDPOINT_ID",
+     "transforms": "unwrap",
+     "transforms.unwrap.type": "io.debezium.transforms.ExtractNewRecordState"
```

Comment:
maybe we can add also `"database.tcpKeepAlive": "true",`? So that by default the connection isn't closed if the connector isn't used for a while. Not sure if add it or not

Reply:
Done.